app/views/profiles/show.html.erb

```diff
@@ -1,15 +1,54 @@
+<%= stylesheet_link_tag 'profiles' %>
+
 <div class="mx-auto md:w-2/3 w-full flex">
   <div class="mx-auto">
     <% if notice.present? %>
       <p class="py-2 px-3 bg-green-50 mb-5 text-green-500 font-medium rounded-lg inline-block" id="notice"><%= notice %></p>
     <% end %>
 
-    <%= render @profile %>
+    <div class="profile-header-container">
+      <%= image_tag(@profile.avatar_url) %>
+      <h1 style="font-size: 24px; font-weight: bold;">
+        <%= User.find_by(id: @profile.user_id).github_username %>
```

Comment:
Now that we've defined the [relationships on the models](https://github.com/GitRepsCommunity/platform/pull/79), we can do this:

```ruby
<%= @profile.user.github_username %>
```

Reply:
Also, in general, we want to avoid directly using the ORM (i.e. `User.find_by(...)`) in a view. It's better practice to define database queries in the controller or in the model then assign the results to a variable.

Reply:
https://www.figma.com/file/jMxsMPkhKyeLo8qm1EQjjZ/Git-Reps-Profile-Page-Mock-Up?type=design&node-id=0-1&mode=design&t=EKlRyzbwzMot9QAS-0